### PR TITLE
Portfolio: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by saruuja**
I made a master branch from the feature branch. I  have done the

### CODE REVIEW

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @saruuja 

Great job so far with your progress 😄 . There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Good commit messages
✅ Good README template


Required Changes ♻️
Check the comments under the review

- [ ] please Add a description summary to your pull request. you can do that by clicking the 3dots and choosing edit. 👇🏾 ;
Screenshot (33)_LI
- [ ]please ensure that your toolbar section is a `header` tag container
- [ ]Remove the commented code for clean code practice
- [ ] please ensure that the `Headline` section is inside a `Section tag` container.
- [ ] let's make your main title(the one with your name) an `h1 tag`.


**Optional suggestions**

- [optional] You can add your name in the place of `my logo` if you don't have one yet
- [ optional] please use reasonable class names
- [ optional] please next time do not create a new pull request or new branch unless the previous one has been approved by a code reviewer.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by saruuja** [comment]
I made all Required Changes and  the comments under the review like
. I change the title to h1 
. I add a section in the headline part
. Also, I change the font and size of my logo like Figma one

File: style.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
```
[the remark this answers is not in the document]
Comment (zieeco) [reply style.css]: **[REQUIRED CHANGES]**

- As required by the previous, though you made some changes, however, your design is not yet close to the Figma design in terms of font-weight. please, compare your design to that of the Figma and adjust your styles to match the design.

**Comment by saruuja** [comment]
I make my logo font-weight as the Figma design 700

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @saruuja ,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

- [ ] [OPTIONAL] Kindly add descriptive pull request message, This helps other programmers who want to take a look at your code understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**